recipe/meta.yaml

```diff
@@ -50,7 +52,6 @@ about:
   doc_url: https://ipywidgets.readthedocs.io/en/latest/
   dev_url: https://github.com/ipython/ipywidgets
   doc_source_url: https://github.com/jupyter-widgets/ipywidgets/blob/master/docs/source/index.rst
```

Comment:
AFAK we keep doc_url instead of doc_source_url. Otherwise it is overspecified and hence linter is not happy.